Formula/g/go-librespot.rb

```diff
@@ -0,0 +1,61 @@
+class GoLibrespot < Formula
+  desc "Yet another open source Spotify client, written in Go"
+  homepage "https://github.com/devgianlu/go-librespot"
+  url "https://github.com/devgianlu/go-librespot/archive/refs/tags/v0.3.3.tar.gz"
+  sha256 "15678b4d6954b381fb9c5a84aca2b0d7706e4d0b2c3644bafcb79cea6d31064c"
+  license "GPL-3.0-only"
+
+  head "https://github.com/devgianlu/go-librespot.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Why do we need this? Does the default livecheck (e.g. `brew livecheck --debug --autobump go-librespot`) return the wrong results without this?

Reply:
This was a leftover, my bad.